qiskit_experiments/library/characterization/t2ramsey_analysis.py

```diff
@@ -151,7 +151,7 @@ def _format_plot(ax, unit, fit_result, conversion_factor):
 
         # Output unit is 'sec', regardless of the unit used in the input
         result_t2 = AnalysisResultData(
-            "T2",
+            "T2*",
```

Comment:
```suggestion
            "T2star",
```
incase special characters become a problem for DB